lighthouse-core/test/audits/byte-efficiency/offscreen-images-test.js

```diff
@@ -190,4 +237,70 @@ describe('OffscreenImages audit', () => {
       assert.equal(auditResult.items[0].url, 'a');
     });
   });
+
+  it('finds images loaded before last long task (Lantern)', () => {
```

Comment:
seems like title of this one is a bit off when we're finding a length of 0 :) which case is this meant to be testing?